ATM90E32 advanced features (#3573)
OttoWinter/esphomedocs: components/sensor/atm90e32.rst

    :ref:`Sensor <config-sensor>`.
  - **power_factor** (*Optional*): Use the power factor value on this phase. All options from
    :ref:`Sensor <config-sensor>`.

>   - **phash_angle** (*Optional*): Use the phase angle value on this phase in degrees. All options from

Reviewer: ```suggestion
  - **phase_angle** (*Optional*): Use the phase angle value on this phase in degrees. All options from
```

- **phase_c** (*Optional*): The configuration options for the 3rd phase. Same options as 1st phase.
- **frequency** (*Optional*): Use the frequenycy value calculated by the meter. All options from
  :ref:`Sensor <config-sensor>`.
- **peak_current_signed** (*Optional*): Control the peak current output as signed or absolute. All options from

>   :ref:`Sensor <config-sensor>`.

Reviewer: This doesnt exist in the code PR?

Author: Hmm, I will check what happened, it is on one of my local repos, I may have added it to the wrong local branch.

Author: Thanks for the eagle eyes on this. Ok I have cherry picked the missing piece from the local copy.

> - **peak_current_signed** (*Optional*): Control the peak current output as signed or absolute. All options from
>   :ref:`Sensor <config-sensor>`.

Reviewer: ```suggestion
- **peak_current_signed** (*Optional*, boolean): Control the peak current output as signed or absolute. Defaults to ``false``.
```
